### PR TITLE
Stripe Refactor

### PR DESCRIPTION
**Pull request by lemieux**
This makes the widget use the appName and iconUrl from the config.

Also, I remove esformatter-semicolons because it wasn't working with jsx.

### CODE REVIEW

File: src/frame/js/components/Action.jsx [hunk 1]
```diff
@@ -200,7 +201,9 @@ export default connect(({ui: {text}, user, config}) => {
         user,
         actionPaymentCompletedText: text.actionPaymentCompleted,
         integrations: config.integrations,
-        stripe: config.stripe
+        stripe: config.stripe,
```
Comment (mspensieri) [inline src/frame/js/components/Action.jsx]: still exists or no?

File: src/frame/js/components/Action.jsx [hunk 2]
```diff
@@ -139,8 +140,8 @@ export class ActionComponent extends Component {
                                        email={ user.email }
                                        amount={ amount }
                                        currency={ currency.toUpperCase() }
-                                       name={ stripeAccount.appName }
-                                       image={ stripeAccount.iconUrl }
```
Comment (mspensieri) [inline src/frame/js/components/Action.jsx]: Is there a service or action that was fetching this data that should be deleted also?

File: src/frame/js/components/Action.jsx (hunk 1 above)
Comment (lemieux) [inline src/frame/js/components/Action.jsx]: No it doesn't, info is in the config.integrations array now.

File: src/frame/js/components/Action.jsx (hunk 2 above)
Comment (lemieux) [inline src/frame/js/components/Action.jsx]: Yes, this https://github.com/smooch/smooch-js/pull/582/commits/89faf6ba83303f89c2135857409b48bb8e79bf6e

File: src/frame/js/components/Action.jsx (hunk 1 above)
Comment (mspensieri) [inline src/frame/js/components/Action.jsx]: kill this prop then?

Comment (lemieux) [inline src/frame/js/components/Action.jsx]: oh I thought it was dead

**Pull request merged**